parsers/US_SPP.py

```diff
@@ -233,9 +243,7 @@ def fetch_load_forecast(
         dt = target_datetime
     else:
         dt = parser.parse(target_datetime)
-    LOAD_URL = "https://marketplace.spp.org/file-api/download/mtlf-vs-actual?path=%2F{0}%2F{1:02d}%2F{2:02d}%2FOP-MTLF-{0}{1:02d}{2:02d}0000.csv".format(
-        dt.year, dt.month, dt.day
-    )
+    LOAD_URL = f"{US_PROXY}/chart-api/load-forecast/asFile?{HOST_PARAMETER}"
```

Comment:
Was this URL changed on purpose?

Reply:
Yeah both were updated as the old one did not return anything